eslint.config.js

```diff
@@ -137,14 +130,6 @@ module.exports = [
             "spaced-comment": ["error", "always", { exceptions: ["-"] }],
             strict: ["error", "global"],
             "use-isnan": "error",
-            "valid-jsdoc": [
```

Comment:
This rule is deprecated and had issues with the comments I added. We can replace with https://github.com/gajus/eslint-plugin-jsdoc later.